homeassistant/components/unifi/sensor.py

```diff
@@ -224,6 +234,46 @@ class UnifiSensorEntityDescription(
         unique_id_fn=lambda controller, obj_id: f"outlet_power-{obj_id}",
         value_fn=lambda _, obj: obj.power if obj.relay_state else "0",
     ),
+    UnifiSensorEntityDescription[Devices, Device](
+        key="SmartPower ac power budget",
```

Comment:
```suggestion
        key="SmartPower AC power budget",
```
Is SmartPower a UniFi name?